src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put.ts

```diff
@@ -8,10 +9,12 @@ export const JiraConnectEnterpriseAppPut = async (
 	try {
 		req.log.debug("Received Jira Connect Enterprise App PUT request");
 
-		// TODO: Add logic for updating apps
+		await GitHubServerApp.updateGitHubApp(req.body);
```

Comment:
Interested to hear people's thoughts here: should we do a read first (get current entry in GHSA), compare the objects, and if there's a difference, then update? I figured we're only updating one entry so should be pretty fast/cheap but am open to opinions.

Reply:
**IMPORTANT**

I think we need to rely on the `gitHubSeverAppMiddlware` to provide the `uuid/gitHubAppId` info, instead of relying on the `req.body`, as user can feed in any `uuid` they want, might accidentally override others.

Reply:
Okay, `gitHubSeverAppMiddlware` is not available, since this is the `jira` route. Maybe we can you please explain how is the `uud` verified that it belongs to the user?